var/spack/repos/builtin/packages/charliecloud/package.py

```diff
@@ -13,6 +13,19 @@ class Charliecloud(MakefilePackage):
     url      = "https://github.com/hpc/charliecloud/releases/download/v0.9.10/charliecloud-0.9.10.tar.gz"
     git      = "https://github.com/hpc/charliecloud.git"
 
+    depends_on('python@3.4:', type=('build', 'run'))
+
+    # experimental builder (ch-grow)
+    variant('ch-grow', default=False, description='Bundle dependencies for unprivileged builder')
```

Comment:
I would call it `chgrow` or `ch_grow`. Variants can be disabled with `-`, so a dash in the variant name can be misleading.